solutions/cses-1081.mdx

```diff
@@ -145,3 +147,24 @@ int main() {
 	}
 }
 ```
+</CPPSection>
+<PySection>
+```py
+MAXN = int(1e6+5)
```

Comment:
this constant isn't needed, just use `n` instead

Reply:
The constant is needed. Kevin do you even read the problem before writing PR comments?

Reply:
oh wait yeah i'm dumb lol

Reply:
but change it to just 1e6

Reply:
1e6+1? And I don't think it matters particularly

Reply:
i can change it for you if you want

Reply:
It won't work with just 1e6

Reply:
i know, but MAX N is still 1e6, so we use 1e6 and call MAXN + 1

Reply:
erm isn't this 1e5?